client/landing/gutenboarding/hooks/use-domain-suggestions.ts

```diff
@@ -1,27 +1,35 @@
 /**
  * External dependencies
  */
-import { useSelect } from '@wordpress/data';
 import { useDebounce } from 'use-debounce';
+import { useSelect } from '@wordpress/data';
+import { useI18n } from '@automattic/react-i18n';
 
 /**
  * Internal dependencies
  */
 import { DOMAIN_SUGGESTIONS_STORE } from '../stores/domain-suggestions';
 import { STORE_KEY as ONBOARD_STORE } from '../stores/onboard';
+import { USER_STORE } from '../stores/user';
 import { DOMAIN_SUGGESTION_VENDOR, selectorDebounce } from '../constants';
 import { useCurrentStep } from '../path';
 
 export function useDomainSuggestions( { searchOverride = '', locale = 'en' } ) {
+	const { __ } = useI18n();
 	const { siteTitle, siteVertical, domainSearch } = useSelect( ( select ) =>
 		select( ONBOARD_STORE ).getState()
 	);
+	const currentUser = useSelect( ( select ) => select( USER_STORE ).getCurrentUser() );
 	const currentStep = useCurrentStep();
 	const prioritisedSearch = searchOverride.trim() || domainSearch.trim() || siteTitle;
 	let searchVal;
 
 	if ( currentStep !== 'IntentGathering' ) {
-		searchVal = prioritisedSearch || siteVertical?.label.trim() || '';
+		searchVal =
+			prioritisedSearch ||
+			siteVertical?.label.trim() ||
+			currentUser?.username ||
+			__( 'My new site' );
```

Comment:
Used this as a fallback query to add a touch of personality but any other suggestion is welcome. 
Hint: we already have translations for ["new site"](https://translate.wordpress.com/projects/wpcom/-all-translations/230582/).